cernopendata/jsonschemas/records/record-v1.0.0.json

```diff
@@ -304,42 +362,53 @@
     "prepublication": {
       "properties": {
         "date": {
+          "description": "The prepublication date, based on ISO 8601 (YYYY-MM-DD)",
```

Comment:
I like the notion of format `YYYY-MM-DDD` it may be useful to repeat this elsewhere...  as you do with `8TeV`... then the documentation is more readable standalone even without seeing record examples